app/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.github.muellerma.prepaidbalance">
 
     <uses-permission android:name="android.permission.CALL_PHONE" />
     <uses-permission android:name="android.permission.READ_PHONE_STATE" />
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"
+        android:maxSdkVersion="28"
+        tools:ignore="ScopedStorage" />
```

Comment:
> Scoped storage limits app access to external storage. In Android 11 or higher, apps targeting API 30 or higher must use scoped storage. 

from https://source.android.com/docs/core/storage/scoped?hl=en

Can you also implement ScopedStorage then? Another option would be to open a share dialog with the file and let the user choose the target, e.g. a mail client. That's probably less to implement.

Reply:
> > Scoped storage limits app access to external storage. In Android 11 or higher, apps targeting API 30 or higher must use scoped storage.
> 
> from https://source.android.com/docs/core/storage/scoped?hl=en
> 
> Can you also implement ScopedStorage then? Another option would be to open a share dialog with the file and let the user choose the target, e.g. a mail client. That's probably less to implement.

should clicking the recycler view item implement this share dialog?